ext/standard/file.h

```diff
@@ -21,19 +21,6 @@
 
 PHP_MINIT_FUNCTION(file);
 PHP_MSHUTDOWN_FUNCTION(file);
-
-PHPAPI PHP_FUNCTION(fclose);
-PHPAPI PHP_FUNCTION(feof);
-PHPAPI PHP_FUNCTION(fread);
-PHPAPI PHP_FUNCTION(fgetc);
-PHPAPI PHP_FUNCTION(fgets);
-PHPAPI PHP_FUNCTION(fwrite);
-PHPAPI PHP_FUNCTION(fflush);
-PHPAPI PHP_FUNCTION(rewind);
-PHPAPI PHP_FUNCTION(ftell);
-PHPAPI PHP_FUNCTION(fseek);
-PHPAPI PHP_FUNCTION(fpassthru);
```

Comment:
I don't think these should be dropped, because they're used for aliases. It looks like it still works because we generate forward-declaration for aliases in arginfo.h, but we probably shouldn't be doing that.

Reply:
yeah, I wasn't sure about this, but I felt ok after the tests passed... Nevertheless, I reverted this change.